### PR TITLE
added collapsibles to procedure section

### PR DESCRIPTION
**Pull request by mohamedwane1**

### CODE REVIEW

File: src/main/webapp/statistics.html [hunk 1]
```diff
@@ -152,10 +158,24 @@ <h2>Procedures</h2>
             very little data on them
           </li>
         </ul>
-    </ul>
+        <script>
```
Comment (bhaamati) [inline src/main/webapp/statistics.html:111]: Would it maybe be better if the javascript was moved to another file?

Comment (bhaamati) [reply src/main/webapp/statistics.html:111]: There are actually multiple blocks of javascript in the same file. Any reason why they aren't consolidated?

Comment (mohamedwane1) [reply src/main/webapp/statistics.html:111]: That's a good point! I moved all of the javascript for the charts into a new file and I'll work on moving the collapsible as well.

**Review by ekwkk — APPROVED** [review]
LGTM!

**Pull request merged**